source/unix/main.cpp

```diff
@@ -421,33 +421,36 @@ void nst_movie_stop() {
 	}
 }
 
-void nst_play() {
+bool nst_play() {
 	// Play the game
-	if (playing || !loaded) { return; }
```

Comment:
FYI, the !loaded conditional is not needed, as it is explicitly set to 1 before the only place where this function is called.